alor-broker/Astras-Trading-UI
# Issue #258: Added risks display for forward market portfolio

Fixes #258  

# Description

Added risks display for forward market profiles

# Testing

- Select portfolio for forward market
- Check portfolio summary

# Risk

No major changes

# Do you agree with those?
